docs/user_documentation.md

```diff
@@ -783,6 +783,45 @@ The following are fields on the Data Source Object:
 
 -   **Reset Data Source (Console Button):** Disables and resets data source to retrieve data from specified Start Date (Depending on import profile settings, enabling this data source after a reset could duplicate data in the Workspace)
 
+### Data Source Specific Settings
+
+This section contains additional settings which are not associated with specific
+Relativity Fields. The settings described here are common across all Data Source
+Types. Type-specific settings are documented under their respected Data Source 
+sections.
+
+-   **Password Bank** Used to specify known passwords to attempt
+    while encountering protected native files. Multiple passwords 
+    can be separated by the pipe character, |. Passwords containing
+    the pipe character are supported through escaping the pipe character
+    with a second pipe. Pipes are always escaped left to right.
+
+    > **Example Password Bank:** passw0rd|Trace1234!|aaa|bb|cccc||dd||eee|||ff|||ggg||||hhh|||||
+    >
+    > Yields the following passwords:
+    >
+    > * passw0rd
+    > * Tracer1234!
+    > * aaa
+    > * bb
+    > * cccc|dd|eee|
+    > * ff|
+    > * ggg||hhh||
+
+- **Extraction Thread Count:** The number of documents to extract in parallel.
```

Comment:
When would a user adjust this setting? I'm not sure we should document it.

Reply:
It would be on our recommendation in order to attempt to increase extraction speed.

If a user found that they are falling behind on Enrichment/Ingestion, they could tweak 1) the number of TraceWorkerAgents they have and 2) the number of threads spawned by each agent.